packages/nextcloud/pubspec.yaml

```diff
@@ -6,37 +6,37 @@ environment:
   sdk: '>=3.1.3 <4.0.0'
 
 dependencies:
-  built_collection: ^5.1.1
-  built_value: ^8.6.3
-  collection: ^1.17.1
-  crypto: ^3.0.3
-  crypton: ^2.2.0
+  built_collection: ^5.0.0
+  built_value: ^8.0.0
+  collection: ^1.0.0
+  crypto: ^3.0.0
+  crypton: ^2.0.0
   dynamite_runtime:
     git:
       url: https://github.com/nextcloud/neon
       path: packages/dynamite/dynamite_runtime
-  intl: ^0.18.1
-  json_annotation: ^4.8.1
-  meta: ^1.9.1
-  universal_io: ^2.2.2
-  version: ^3.0.2
-  xml: ^6.3.0
-  xml_annotation: ^2.2.0
+  intl: ^0.18.0
+  json_annotation: ^4.0.0
+  meta: ^1.0.0
+  universal_io: ^2.0.0
+  version: ^3.0.0
+  xml: ^6.0.0
+  xml_annotation: ^2.1.0
 
 dev_dependencies:
-  build_runner: ^2.4.6
-  built_value_generator: ^8.6.3
-  coverage: ^1.6.4
+  build_runner: ^2.0.0
+  built_value_generator: ^8.0.0
```

Comment:
We'll need at least 8.5 for the string interpolation issue I fixed upstream
https://pub.dev/packages/built_value_generator/changelog#850